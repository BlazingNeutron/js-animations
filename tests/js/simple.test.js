/**
 * @jest-environment jsdom
 */
const setup = require('../../src/js/simple');

document.body.innerHTML =
    '<div class="content"/>';

test('contains a canvas', () => {
  setup();
  let canvas = document.querySelector("canvas");
  expect(canvas).not.toBeNull();
});

test('renders empty canvas correctly', () => {
  setup();
  let canvas = document.querySelector("canvas");
  expect(canvas).toMatchSnapshot();
});