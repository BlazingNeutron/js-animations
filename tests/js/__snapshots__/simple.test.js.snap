// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders empty canvas correctly 1`] = `
<canvas
  style="position: fixed; top: 0px; left: 0px; width: 100%; height: 100%;"
/>
`;
